### DISCUSSION TITLE
[Feature Request] Add a workflow trigger for environment variable update

### DISCUSSION THREAD

**Discussion by kehtabp** [community/community#73422] [Actions]
### Select Topic Area

Product Feedback

### Body

It would be great if triggering workflow was not necessary after environment variable is updated

**Comment by onurtufekci34** [comment]
Thank you for your feedback regarding the process of triggering a workflow after updating an environment variable. Your input is valuable for improving workflow automation processes. While I don't have the real-time capability to make changes to specific products or platforms, you might consider sharing this feedback directly with the platform or service you're using for workflows and automation. They often have feedback channels or support forums where you can suggest improvements or inquire about any possible updates to streamline the process you mentioned. This input can help drive future improvements in the platform's functionality.
